.github/workflows/auto-merge/automerge

```diff
@@ -124,9 +115,8 @@ def main():
     if term:
         sys.exit(0)
 
-    time.sleep(10)  # sleep, then comment
+    time.sleep(10)  # sleep 10s, then try auto-merge
     auto_merge(number, sha)
 
 
-if __name__ == "__main__":
```

Comment:
Curious, why was this removed?

Reply:
try to make it a pure script lol

Reply:
What is gained by this change?  That check doesn't preclude the code from being used like a script, right?  It seems like if anything this change keeps the code from being reusable in other workflows as a module.  If this change didn't fix anything, it makes the code less reusable for no benefit.

Reply:
hmm, actually that script is not reusable before, I just want to unblock the auto-merge stuff earlier. 
As you request, let me refactor the whole thing. Give me some minutes~

Reply:
Yes, you're right, it needs a bit more work than just the check.  Can clean this up in a followup.